components/KeyManagerPermissions/KeyManagerPermissions.tsx

```diff
@@ -68,19 +105,52 @@ const KeyManagerPermissions: React.FC = () => {
         </div>
       </div>
       <div className="columns">
-        <div className="column">
+        <div className="column is-half">
+          <h2 className="mb-2 subtitle is-5">
+            Manage Contract Ownership
+            <a
+              href="https://docs.lukso.tech/standards/universal-profile/lsp6-key-manager/#permissions"
+              target="_blank"
+              rel="noreferrer"
+              className="ml-2 has-text-info-dark"
+            >
+              ↗
+            </a>
+          </h2>
           <PermissionsBtns
             permissions={['CHANGEOWNER', 'ADDCONTROLLER', 'EDITPERMISSIONS']}
             color={'is-orange-dark'}
             decodedPermissions={decodedPermissions}
             handlePermissionClick={handlePermissionClick}
           />
+          <h2 className="mb-2 subtitle is-5">
+            Manage Extensions
+            <a
+              href="https://docs.lukso.tech/standards/universal-profile/lsp6-key-manager/#permissions"
+              target="_blank"
+              rel="noreferrer"
+              className="ml-2 has-text-info-dark"
+            >
+              ↗
+            </a>
+          </h2>
           <PermissionsBtns
             permissions={['ADDEXTENSIONS', 'CHANGEEXTENSIONS']}
             color={'is-orange'}
             decodedPermissions={decodedPermissions}
             handlePermissionClick={handlePermissionClick}
           />
+          <h2 className="mb-2 subtitle is-5">
+            Manage Universal Receivers
+            <a
+              href="https://docs.lukso.tech/standards/universal-profile/lsp6-key-manager/#permissions"
```

Comment:
what's the point of having all of these arrow going to the same link? That's confusing to me. I'll simply remove as it adds no value. There is already a link at the top of that page.